src/configFile.ts

```diff
@@ -0,0 +1,73 @@
+import { parse } from "toml";
+import { readFile } from "fs";
+import findUp from "find-up";
+interface foremanConfig {
+  tools: {
+    [tool_name: string]: foremanTool
+  };
+}
+
+interface foremanTool {
+  source?: string;
+  github?: string;
+  gitlab?: string;
+  version: string;
+}
+
+const MANIFEST = "foreman.toml";
+
+function checkSameOrgToolSpecs(manifestContent: foremanConfig): boolean {
+  const tools = manifestContent.tools;
+  if (tools == null) {
+    throw new Error("Tools section in Foreman config not found");
+  }
+
+  const orgs: string[] = [];
+  for (const tool_name in tools) {
+    const tool_spec = tools[tool_name];
+    let source = tool_spec["source"];
+    if (source == null) {
+      source = tool_spec["github"];
+    }
+    if (source == null) {
+      continue;
+    }
+
+    const source_array = source.split("/");
+    const org = source_array[0];
+
+    if (org == null) {
+      throw new Error(
+        `Org not found in tool spec definition for: ${tool_name}`
+      );
+    }
+    orgs.push(org.toLowerCase());
+  }
+  if (orgs.length == 0) {
+    return true;
+  }
+  return orgs.every(val => val === orgs[0]);
```

Comment:
This isn't actually quite what I meant. I think we need to figure out what org _this github action_ is running on and enforce _that_.

It shouldn't just enforce that I'm getting all my stuff from the same 3rd party; it should enforce that I'm getting all my stuff from the same org I'm running CI in right now.